cirq/contrib/acquaintance/permutation.py

```diff
@@ -234,15 +234,26 @@ def get_logical_operations(operations: ops.OP_TREE,
 
 class ExpandPermutationGates(optimizers.ExpandComposite):
     """Decomposes any permutation gates other SwapPermutationGate."""
```

Comment:
update doc since (which has a grammar bug btw) to describe that this sometimes keeps the SwapPermutationGates

Reply:
Done